### ISSUE TITLE
onConnect shouldn't demand arity

### ISSUE THREAD

**Issue by antonsamper** [valtlfelipe/hapi-sequelizejs#2]
Currently, the `onConnect` property validates against `arity(1)` but i don't think this is required because the database instance may not be required inside the `onConnect` function.

###### Currently
```javascript
{
  onConnect: Joi.func().arity(1)
}
```

###### Proposed
```javascript
{
  onConnect: Joi.func()
}
```

What do you think?

**Comment by valtlfelipe** [comment]
Hi @antonsamper!

It is true, it is not necessary. I will change the validation.

**Comment by valtlfelipe** [comment]
Fixed in a9d0965.
